web/src/components/map.js

```diff
@@ -337,18 +345,24 @@ export default class Map {
     return this;
   }
 
-  onCountryMouseMove(arg) {
-    if (!arg) return this.countryMouseMoveHandler;
-    else this.countryMouseMoveHandler = arg;
-    return this;
-  }
-
   onCountryMouseOut(arg) {
```

Comment:
Cam you also rename this one for consistency?

Reply:
Renamed `onCountryMouseOut` to `onZoneMouseOut`. Should `onCountryClick` and `onCountryMouseOver` be renamed to `onZone...`?